server/s3.js

```diff
@@ -1,37 +1,46 @@
-require('dotenv').config();
+require("dotenv").config();
 const AWS = require("aws-sdk");
 const fs = require("fs");
 
-const bucketName = process.env.AWS_BUCKET_NAME; 
-const secretKey =  process.env.AWS_SECRET_ACCESS_KEY;
-const region = process.env.AWS_REGION; 
-const accessKey = process.env.AWS_ACCESS_KEY_ID; 
+const bucketName = process.env.AWS_BUCKET_NAME;
+const secretKey = process.env.AWS_SECRET_ACCESS_KEY;
+const region = process.env.AWS_REGION;
+const accessKey = process.env.AWS_ACCESS_KEY_ID;
 
 const s3 = new AWS.S3({
-  accessKey, 
+  accessKey,
   secretKey,
   region
 });
 
+// upload one file to S3
```

Comment:
I do like seeing you comment, I think you've done a better job at just making the code explain itself. In this case, it's safe to remove these comments.